.github/workflows/development-project-automation.yml

```diff
@@ -0,0 +1,76 @@
+name: Development project automations
+on:
+  workflow_call:
+    inputs:
+      project-number:
+        required: true
+        description: The project number from the project's url
+        type: number
+      resource-node-id:
+        required: true
+        description: The node id of the issue or PR to act upon
+        type: string
+      event-action:
+        required: true
+        description: The event action that triggered the original workflow
+        type: string
+      event-name:
+        required: true
+        description: The name of the event that triggered the original workflow
+        type: string
+        default: EventStore
+    secrets:
+      token:
+        required: true
+        description: The token to use. It must have org:write access
+
+env:
+  PROJECT_ID: ${{ inputs.project-number }}
+jobs:
+  milestone_added_to_issue:
+    name: milestone_added_to_issue
+    runs-on: ubuntu-latest
+    if: inputs.event-name == 'issues' && inputs.event-action == 'milestoned'
+    steps:
+      - name: 'Add issue to board'
+        uses: leonsteinhaeuser/project-beta-automations@v1.0.2
+        with:
+          gh_token: ${{ secrets.token }}
```

Comment:
**QUESTION:** Cannot we use `GH_TOKEN` here?

Reply:
We can't because the token used for board automation has to have `org:write` access.
The calling workflow will pass through the correct token from the org secrets.